specs/Picker-API_spec.md

```diff
@@ -0,0 +1,246 @@
+    }
+    ```
+
+## Show single folder picker
+    ```c#
+    var picker = StorageItemPicker::CreateFolderPicker();
+    Windows.Storage.StorageFolder folder = await picker.PickSingleFolderAsync();
+    ```
+
+
+## Show multi folder picker
+    ```c#
+    var picker = StorageItemPicker::CreateFolderPicker();
+    IReadOnlyList<Windows.Storage.StorageFolder> folders = picker.PickMultipleFoldersAsync();
+    ```
+
+# Remarks
+
+<!-- TEMPLATE
+
+    Explanation and guidance that doesn't fit into the Examples section.
+
+    APIs should only throw exceptions in exceptional conditions; basically,
+    only when there's a bug in the caller, such as argument exception.  But if for some
+    reason it's necessary for a caller to catch an exception from an API, call that
+    out with an explanation either here or in the Examples
+-->
+
+# API Notes
+
+<!-- TEMPLATE
+
+    Option 1: Give a one or two line description of each API (type and member),
+        or at least the ones that aren't obvious from their name. These
+        descriptions are what show up in IntelliSense. For properties, specify
+        the default value of the property if it isn't the type's default (for
+        example an int-typed property that doesn't default to zero.) 
+        
+    Option 2: Put these descriptions in the below API Details section,
+        with a "///" comment above the member or type. 
+-->
+
+# API Details
+
+<!-- TEMPLATE
+
+    The exact API, in MIDL3 format (https://docs.microsoft.com/en-us/uwp/midl-3/)
+    when possible, or in C# if starting with an API sketch.  GitHub's markdown
+    syntax formatter does not (yet) know about MIDL3, so use ```c# instead even
+    when writing MIDL3.
+
+    Example:
+
+    ```c# (but really MIDL3)
+    namespace Microsoft.AppModel
+    {
+        /// Represents a package on the host system. See Windows.ApplicationModel.Package for more details
+        runtimeclass Package
+        {
+            /// Returns the current package, or null if the current process is not packaged
+            static Package Current { get; };
+
+            /// Returns the package from the system store with this full name or null if not found
+            static Package GetFromFullName(String fullName);
+
+            /// Returns packages in the given family, by name
+            static Package[] FindByFamilyName(String familyName);
+        }
+    }
+    ```
+-->
+    ```c#
+    namespace Microsoft.Reunion.Storage.Pickers
+    {
+        runtimeclass StorageItemPicker
+        {
+            /// Static constructor
+            static FileOpenPicker CreateFileOpenPicker();
+            static FileSaveFileAsync CreateFileSavePicker();
+            static FolderPicker CreateFolderPicker();
+
+            String StartLocation{ set; }
+            String OkButtonText { set; }
+            String CancelButtonText { set; }
+            Windows.Foundation.Collections.IVector<String> Filters{ get; }
+            PickerViewMode ViewMode{ get; set; }
```

Comment:
`PickerViewMode`: No such type?

Reply:
Windows.Storage.Pickers.PickerViewMode

Reply:
I have’t done AppContainer development in years, but I feel that allowing only these two choices for the view mode would hamper the usability of these pickers as compared to the Win32 ones, which allow five or six different view modes. I would prefer removing the use of this enum, and have the view in the picker window be decided by the user, with all options (or as many as are practical to implement) in the Win32 picker available to choose from.

Reply:
> Windows.Storage.Pickers.PickerViewMode

Could you explain please?

Reply:
Apologies for the `:-1` reaction; that was only intended to be temporary while I wrote my rationale in the followup post. Is there anything I can clarify?

Reply:
Noted. We could introduce a new PickerViewMode with more options and a different namespace.